src/helpers/currencyHelper.ts

```diff
@@ -0,0 +1,33 @@
+var currencySymbols = {
+  'USD': '$', // US Dollar
```

Comment:
That's nice, but thankfully somebody thought about this issue long lime before us and created `toLocaleString`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString
so you can simply use currency provided from API and use it like this:
![image](https://user-images.githubusercontent.com/10283222/222267550-af1eaef7-8980-4511-9088-8b220ad10920.png)

Reply:
the only issue is to set it be ₴ fir UAH and zł for PLN, since those sings will only work if you set it to be in your locale, which isn't quite easy...

Reply:
as an alternative, which seems to work as expected: https://www.npmjs.com/package/currency-formatter